File: cranberri-gui/cranberri-gui-electron/project_templates/java_full_standard/scripts/MyScript.java
package com.example.project1;

import jupiterpi.cranberri.runtime.api.*;
import com.example.project1.shared.*;

@Script
public class MyScript {
    @Setup
    public void setup() {
        IO.log("started!");

        IO.log(MySingleton.MY_CONSTANT);
        MySingleton.mySharedMethod();
        
        IO.disableDebug();
    }

    @Tick
    public void tick() {
        IO.writePin(1, IO.readPin(2));
    }
}

File: cranberri-gui/cranberri-gui-electron/project_templates/kotlin_full/scripts/MyScript.kt
package com.example.project1

import jupiterpi.cranberri.runtime.api.*

@Script
class MyScript {
    @Setup
    fun setup() {
        IO.log("started!")

        IO.log(MY_CONSTANT)
        mySharedMethod()
        
        IO.disableDebug()
    }

    @Tick
    fun tick() {
        IO.writePin(1, IO.readPin(2))
    }
}